pom.xml

```diff
@@ -24,7 +24,7 @@
         <version.embedded.cassandra>4.0.1</version.embedded.cassandra>
```

Comment:
can we look into this if it is possible to update to something more recent? I _think_ that potential problem will be wilth that plugin needing Java 11 / 17 or similar while we are still on 8 but it is worth to at least check what is the situation there.